Avoid:
CMake Error in src/plugins/gpgme/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "elektra-gpgme-objects":

    "-D_FILE_OFFSET_BITS=64"

--- elektra-0.9.14/scripts/cmake/Modules/FindLibGpgme.cmake.orig	2023-08-25 20:22:55.946714270 +0200
+++ elektra-0.9.14/scripts/cmake/Modules/FindLibGpgme.cmake	2023-08-25 20:25:26.262566605 +0200
@@ -44,6 +44,31 @@ if (GPGME_EXECUTABLE)
 		set (LIBGPGME_FOUND TRUE)
 	endif (LIBGPGME_LIBRARIES AND LIBGPGME_CFLAGS)
 
+else (GPGME_EXECUTABLE)
+
+	execute_process (
+		COMMAND pkg-config gpgme --libs
+		RESULT_VARIABLE _return_VALUE
+		OUTPUT_VARIABLE LIBGPGME_LIBRARIES
+		OUTPUT_STRIP_TRAILING_WHITESPACE)
+	execute_process (
+		COMMAND pkg-config gpgme --cflags
+		RESULT_VARIABLE _return_VALUE
+		OUTPUT_VARIABLE LIBGPGME_CFLAGS
+		OUTPUT_STRIP_TRAILING_WHITESPACE)
+	execute_process (
+		COMMAND pkg-config gpgme --modversion
+		RESULT_VARIABLE _return_VALUE
+		OUTPUT_VARIABLE LIBGPGME_VERSION
+		OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+	if (NOT LIBGPGME_CFLAGS AND NOT _return_VALUE)
+		set (LIBGPGME_CFLAGS " ")
+	endif (NOT LIBGPGME_CFLAGS AND NOT _return_VALUE)
+
+	if (LIBGPGME_LIBRARIES AND LIBGPGME_CFLAGS)
+		set (LIBGPGME_FOUND TRUE)
+	endif (LIBGPGME_LIBRARIES AND LIBGPGME_CFLAGS)
 endif (GPGME_EXECUTABLE)
 
 if (LIBGPGME_FOUND)
@@ -55,7 +70,10 @@ if (LIBGPGME_FOUND)
 	# parse include directory from C-Flags
 	string (LENGTH "${LIBGPGME_CFLAGS}" LIBGPGME_CFLAGS_LEN)
 	if (${LIBGPGME_CFLAGS_LEN} GREATER 1)
-		string (REPLACE "-I" "" LIBGPGME_INCLUDE_DIRS "${LIBGPGME_CFLAGS}")
+		string (REGEX REPLACE "-D[A-Za-z0-9_=]* *" ""
+			LIBGPGME_INCLUDE_DIRS
+			"${LIBGPGME_CFLAGS}")
+		string (REPLACE "-I" "" LIBGPGME_INCLUDE_DIRS "${LIBGPGME_INCLUDE_DIRS}")
 		string (REPLACE " " ";" LIBGPGME_INCLUDE_DIRS "${LIBGPGME_INCLUDE_DIRS}")
 
 	endif ()
--- elektra-0.9.14/src/plugins/gpgme/CMakeLists.txt.orig	2023-08-25 20:22:55.950047585 +0200
+++ elektra-0.9.14/src/plugins/gpgme/CMakeLists.txt	2023-08-25 20:27:16.921967112 +0200
@@ -16,6 +16,7 @@ add_plugin (
 	SOURCES ${GPGME_SOURCE_FILES}
 	INCLUDE_DIRECTORIES ${LIBGPGME_INCLUDE_DIRS}
 	LINK_LIBRARIES ${LIBGPGME_LIBRARIES}
+	COMPILE_DEFINITIONS "_FILE_OFFSET_BITS=64"
 	ADD_TEST COMPONENT libelektra${SO_VERSION}-experimental)
 
 if (ADDTESTING_PHASE)
